qp_deblur/__init__.py

```diff
@@ -0,0 +1,58 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from qiita_client import QiitaPlugin, QiitaCommand
+
+from .deblur import deblur
+
+__all__ = ['deblur']
+
+
+# Initialize the plugin
+plugin = QiitaPlugin(
+    'deblur', '0.1.0', 'A greedy deconvolution algorithm based on Illumina '
+    'Miseq/Hiseq error profiles')
+
+# Define the HUMAnN2 command
+req_params = {'seqs-fp': ('artifact', ['Demultiplexed'])}
+opt_params = {
+    # parameters not being passed
+    # output-dir
+    # keep-tmp-files
+    # log-level
+    # log-file
+    # is-worker-thread
+    'ref-fp': ['choice:[""]', ''],
+    'ref-db-fp': ['choice:[""]', ''],
+    'mean-error': ['float', '0.005'],
+    'error-dist': ['string', ('1, 0.06, 0.02, 0.02, 0.01, 0.005, 0.005, '
+                              '0.005, 0.001, 0.001, 0.001, 0.0005')],
+    'indel-prob': ['float', '0.01'],
+    'indel-max': ['integer', '3'],
+    'trim-length': ['integer', '100'],
+    'min-reads': ['integer', '0'],
+    'min-size': ['integer', '2'],
+    'negate': ['boolean', 'True'],
+    'threads-per-sample': ['integer', '1'],
+    'jobs-to-start': ['integer', '1'],
+    'overwrite': ['boolean', 'True']
```

Comment:
It is, cause Qiita creates the folder for us and deblur will fail if is not there. Decided to expose it vs. hardcoding them cause we never know what the future will bring.